### PR TITLE
Boilerplate setup for buildifier and yapf through github actions

### PR DESCRIPTION
**Pull request by ciglass**
Added boilerplate setup for buildifier and yapf through github actions.

### CODE REVIEW

**Comment by ciglass** [comment]
@wstucker let me know if you'd prefer to see this as separate PRs for yapf and for buildifier. I put these together originally because I didn't think that we'd have this many yapf changes. I can abandon this one and put up two different ones if you'd rather see that.

**Comment by guoqilu** [comment]
I'm seeing below error when trying to fix CI failures:
image

any ideas? @wstucker

**Comment by wstucker** [comment]
@guoqilu ABI error probably indicates you are picking up the default GCC on the redhat machine. Are you sure that the newest gcc/llvm is being loaded?

**Comment by guoqilu** [comment]
I definitely not sure. Lol.

I don’t know how to confirm which is the latest GCC I should be picking up

gcc/llvm is it a module or something?

**Comment by wstucker** [comment]
@guoqilu I don't remember the exact path on the LT systems anymore. Check out one of the project's env/env.sh scripts for the path. Its probably something like "source /..../llvm..../activate".

File: unit_test.bzl [hunk 1]
```diff
@@ -1,11 +1,13 @@
-def glob_to_individual_py_tests(files):
-    for name in files:
+"""Helper functions to run python unit tests."""
+
+def glob_to_individual_py_tests(files, test=":test", lib=":lib"):  # buildifier: disable=unnamed-macro
```
Comment (ciglass) [inline unit_test.bzl]: @guoqilu can you please remove the 'test' and 'lib' arguments? This macro is only going to be used in one place so we don't need to make it parameterized.

**Pull request merged**